src/test/forms/formHelpers.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ManagedNumberField } from '../../forms/formHelpers';
+
+test('ManagedNumberField should match snapshot', () => {
+  const wrapper = shallow(
+    <ManagedNumberField
+      format={ () => {} }
+      value={ 0 } />
+  );
+  expect(wrapper).toMatchSnapshot();
+});
+
+test('should set focused state to true on focus if current value of FormInput is positive number', () => {
```

Comment:
Shouldn't `focused` state be set to true any time someone clicks in the field? It's just the resulting value of the field after that focus that's in question, right?

Reply:
I think the idea was just to test that `focused` is true when the field is actually focused, and we do this check separately for zero and nonzero values of the field (since we handle them differently).

Reply:
It's not 0 vs. non-zero values, right? It's for any falsy float-parsed value, right? I don't exactly recall the situation that made this necessary, but it had something to do with `NaN`. @shp54 : do you recall?

Reply:
y'know what, we can always fiddle with it later. Thanks, @snyderc!